railgun/test/integration/suite_test.go

```diff
@@ -121,6 +134,21 @@ func TestMain(m *testing.M) {
 		builder = builder.WithExistingCluster(cluster)
 	}
 
+	fmt.Println("INFO: configuring kubernetes cluster")
+	if clusterVersionStr != "" {
+		clusterVersion, err := semver.Parse(strings.TrimPrefix(clusterVersionStr, "v"))
+		if err != nil {
+			fmt.Fprintf(os.Stderr, "Error: invalid %s provided (%s): %s", ClusterVersionEnvVar, clusterVersionStr, err.Error())
```

Comment:
Why do some prints go to stderr, and some to stdout?

Maybe we should we use stderr only? Note that stdout can be buffered which is a downside.

Reply:
All prints before exiting the program due to an error should be going to os.Stderr, if that's not the case then it's a typo. I'll review this and fix any problems.

Reply:
There are three uses of `os.Stdout` in the file:

- `278,291`: simply ensuring that any stdout that was produced before an error occurred doesn't get swallowed by the error, but sill emitting to the same  stream for consistency
- `336`: this one was a typo from weeks ago, but since I'm here I fixed it: 0f2ad268c563f3ed3e261e531dc604b75f556607